### PR TITLE
Fix layer0 tests

### PR DESCRIPTION
**Pull request by tabman83**
Fixed layer0 tests. Changed test server to comply with .NET Core 3 test guidelines.

Layer1 & 2 tests will be fixed separately.

### CODE REVIEW

File: src/Tests/Bullfrog.Client/Client/BullfrogApi.cs [hunk 1]
```diff
@@ -308,9 +308,6 @@ private void Initialize()
             };
             CustomInitialize();
         }
-        /// <summary>
-        /// Lists names of all configured scale groups.
-        /// </summary>
```
Comment (hudo) [inline src/Tests/Bullfrog.Client/Client/BullfrogApi.cs:313]: Why were comments removed?

File: src/Tests/Bullfrog.Client/Client/BullfrogApi.cs [hunk 2]
```diff
@@ -681,7 +668,7 @@ public async Task<HttpOperationResponse> SetDefinitionWithHttpMessagesAsync(stri
             {
                 _requestContent = SafeJsonConvert.SerializeObject(body, SerializationSettings);
                 _httpRequest.Content = new StringContent(_requestContent, System.Text.Encoding.UTF8);
-                _httpRequest.Content.Headers.ContentType =System.Net.Http.Headers.MediaTypeHeaderValue.Parse("application/json-patch+json; charset=utf-8");
+                _httpRequest.Content.Headers.ContentType =System.Net.Http.Headers.MediaTypeHeaderValue.Parse("application/json; charset=utf-8");
```
Comment (hudo) [inline src/Tests/Bullfrog.Client/Client/BullfrogApi.cs:671]: Can you please give some quick explanation here why was this change done?

File: src/Tests/Bullfrog.Tests/ProbeControllerTests.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@ public class ProbeControllerTests : BaseApiTests
     [Fact, IsLayer0]
     public async Task ProbeEndpointTest()
     {
-        var response = await HttpClient.SendAsync(new HttpRequestMessage(HttpMethod.Get, "http://a/probe"));
+        var response = await HttpClient.SendAsync(new HttpRequestMessage(HttpMethod.Get, "http://test/probe"));
```
Comment (hudo) [inline src/Tests/Bullfrog.Tests/ProbeControllerTests.cs:11]: How does this work, does .net coe test server understands "test" dns name?

File: src/Tests/Bullfrog.Tests/appsettings.Development.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "KEYVAULT_URL": "https://esw-tooling-ci-we.vault.azure.net/"
+  "KEYVAULT_URL": "https://esw-bullfrog-ci-we.vault.azure.net/"
```
Comment (hudo) [inline src/Tests/Bullfrog.Tests/appsettings.Development.json:2]: Do we need to move some keys from tooling to BF KV?

File: src/Tests/Bullfrog.Client/Client/BullfrogApi.cs (hunk 1 above)
Comment (tabman83) [reply src/Tests/Bullfrog.Client/Client/BullfrogApi.cs:313]: This is auto generated by AutoRest.

File: src/Tests/Bullfrog.Client/Client/BullfrogApi.cs (hunk 2 above)
Comment (tabman83) [reply src/Tests/Bullfrog.Client/Client/BullfrogApi.cs:671]: Same here, this is auto generated by AutoRest.

File: src/Tests/Bullfrog.Tests/appsettings.Development.json (hunk 4 above)
Comment (tabman83) [reply src/Tests/Bullfrog.Tests/appsettings.Development.json:2]: As far as I checked the bullfrog KV has everything that is needed.

**Comment by mszymeshop** [comment]
@tabman83 Do I understand correctly that the tests stopped working after .net core 3.1 migration and this is to make them operational again? Was there another reason for this change?

**Pull request merged**